Share/NCShareExtension.swift

```diff
@@ -263,7 +263,10 @@ class NCShareExtension: UIViewController, NCListCellDelegate, NCEmptyDataSetDele
 
         // PROFILE BUTTON
 
-        let image = NCUtility.shared.loadUserImage(for: activeAccount.user, displayName: activeAccount.displayName, urlBase: activeAccount.urlBase)
+        let image = NCUtility.shared.loadUserImage(
+            for: activeAccount.user,
+               displayName: activeAccount.displayName,
+               userUrlBase: String(CCUtility.getUserUrlBase(activeAccount.user, urlBase: activeAccount.urlBase)))
```

Comment:
I don't like this very old func CCUtility.getUserUrlBase(...) I would replace it with a:

appDelegate.user + "-" + URL(string: appDelegate.urlBase).host

and remove the CCUtility.getUserUrlBase(...)  whats do you think ?

Reply:
Agreed. I like to use computed vars for those things..